02-esc/index.js

```diff
@@ -1,7 +1,23 @@
 import { getDataFromAPI } from "./getDataFromAPI.js";
+import { showAllChallenges } from "./showAllChallenges.js";
 
-// getDataFromAPI returns an array with all the API data
-// console.log(await getDataFromAPI());
+const challenge_list = document.querySelector('.challenge-list');
+
+let resFromAPI = await getDataFromAPI();
+
+function sortRatingArray(resFromAPI){ 
+
+  resFromAPI.sort(function(a, b){
+    return b.rating - a.rating
+  });
+    
+  let newArray = [];
+  newArray.push(resFromAPI[0], resFromAPI[1], resFromAPI[2]);
```

Comment:
The following lines of code do the same thing, but are better IMO:

**Option 1**
```js
const newArray = [ resFromAPI[0], resFromAPI[1], resFromAPI[2] ];
```

**Option 2**
```js
const newArray = resFromApi.slice(0, 3);
```